src/main/java/com/strandls/observation/service/Impl/ObservationDataTableServiceImpl.java

```diff
@@ -172,7 +172,7 @@ public Long observationBulkUpload(HttpServletRequest request, ObservationBulkDTO
 							request.getHeader(HttpHeaders.AUTHORIZATION));
 					UserGroupCreateDatatable ugMapping = new UserGroupCreateDatatable();
 					ugMapping.setUserGroupIds(userGroupIds);
```

Comment:
![Screenshot from 2022-10-14 11-02-07](https://user-images.githubusercontent.com/30981535/195769628-b95f7725-ac7c-4ebc-b813-8cbb057300f0.png)
The above info is not set in this flow, pls add them as well